### PR TITLE
OnClientCheckResponse & SendClientCheck on the english docs

### PR DESCRIPTION
**Pull request by MySekwel**

### CODE REVIEW

File: docs/scripting/callbacks/OnClientCheckResponse.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: OnClientCheckResponse
+description: This callback is called when a SendClientCheck request comletes
+tags: []
+---
+
+## Description
+
+This callback is called when a SendClientCheck request comletes.
```
Comment (Alasnkz) [inline docs/scripting/callbacks/OnClientCheckResponse.md:9]:
```suggestion
This callback is called when a SendClientCheck request completes.
```

File: docs/scripting/callbacks/OnClientCheckResponse.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+{
+    SendClientCheck(playerid, 0x48, 0, 0, 2);
+    return 1;
+}
+
+public OnClientCheckResponse(playerid, actionid, memaddr, retndata)
+{
+    if(actionid == 0x48)
+    {
+        print("The player's client response is 0x48");
```
Comment (Alasnkz) [inline docs/scripting/callbacks/OnClientCheckResponse.md:35]: The response isn't 0x48, the response would be what retndata has.

File: docs/scripting/functions/SendClientCheck.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+{
+    SendClientCheck(playerid, 0x48, 0, 0, 2);
+    return 1;
+}
+
+public OnClientCheckResponse(playerid, actionid, memaddr, retndata)
+{
+    if(actionid == 0x48)
+    {
+        print("The player's client response is 0x48");
```
Comment (Alasnkz) [inline docs/scripting/functions/SendClientCheck.md:38]: ditto

File: docs/scripting/functions/SendClientCheck.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    }
+    return 1;
+}
+```
+
+## Notes
+
+:::warning
+
+This callback is only called when it is in a filterscript.
```
Comment (Alasnkz) [inline docs/scripting/functions/SendClientCheck.md:48]: You should probably link to OnClientCheckResponse here.

**Review by daddyDOT — APPROVED** [review]

**Review by Southclaws — APPROVED** [review]

**Pull request merged**